php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -353,6 +353,16 @@ class NDB_Menu_Filter extends NDB_Menu
         if (isset($_REQUEST['keyword'])) {
             // It was included in the request, so save it.
             $key = $_REQUEST['keyword'];
+            // Check if multiple keyword search
+            if ($_REQUEST['test_name'] == 'datadict'
```

Comment:
This shouldn't be in the base class. If datadict needs to do something special, it should override the function.

Reply:
(same with all the instances below.)

Reply:
Awesome it makes total sense to do that! Will do.